src/export/bops/index.ts

```diff
@@ -1,30 +1,33 @@
 import isEmpty from "lodash.isempty";
 import isNil from "lodash.isnil";
 
+import { Passport } from "../../models/passport";
 import { getResultData } from "../../models/result";
 import { sortBreadcrumbs } from "../../models/session/logic";
-import {
-  BOPSFullPayload,
+import type {
   Breadcrumbs,
-  ComponentType,
-  DEFAULT_APPLICATION_TYPE,
   EnhancedGISResponse,
   FileTag,
-  flatFlags,
   FlowGraph,
-  GOV_PAY_PASSPORT_KEY,
   GovUKPayment,
-  LooseBreadcrumbs,
-  LooseFlowGraph,
-  Passport,
+  Passport as IPassport,
   QuestionAndResponses,
   QuestionMetaData,
   Response,
   ResponseMetaData,
+  SiteAddress,
+  UserRole,
+} from "../../types";
+import {
+  BOPSFullPayload,
+  ComponentType,
+  DEFAULT_APPLICATION_TYPE,
+  flatFlags,
+  GOV_PAY_PASSPORT_KEY,
   USER_ROLES,
 } from "../../types";
 
-const bopsDictionary = {
+const bopsDictionary: { [Property in keyof BOPSFullPayload]?: string } = {
```

Comment:
A better/tighter approach here might just be to use `as const` for this variable.

Instead of casting to `string` on line 145 we can then just use a type guard instead. Something like -

```ts
const isValidKey = (fn: unknown): fn is keyof typeof bopsDictionary => !Object.values(bopsDictionary).find((val) => val === fn)
```